src/blockdata/block.rs

```diff
@@ -527,6 +580,20 @@ mod tests {
 
         assert_eq!(header.bits, BlockHeader::compact_target_from_u256(&header.target()));
     }
+
+    #[test]
+    fn soft_fork_signalling_test() {
```

Comment:
By convention Rust tests should not include the word 'test' in the name (I know this is violated many times in our stack), but none the less thought I'd mention it :)